community/runit/runit.initd

```diff
@@ -8,7 +8,10 @@ description="starts and monitors a collection of runsv(8) processes"
 retry="SIGHUP/5"
 
 depend() {
-    need localmount
-    after firewall
+	need localmount
+	after firewall
+	if [ -x /sbin/socklog ]; then
+		provide logger
+	fi
```

Comment:
`runit` runs the `socklog` service (`socklog` does not have an initd)

without this in the `runit` initd - busybox's syslog will be started (even if it is disabled in `rc-update`) - because `crond` requires `logger` & the only thing that provides it is busybox syslog `initd`

if `/etc/init.d/runitd` provides `logger` (if `socklog` is detected) - busybox syslog correctly does not get started by `openrc`.